client/landing/stepper/declarative-flow/internals/steps-repository/link-in-bio-intro/intro.tsx

```diff
@@ -0,0 +1,21 @@
+import { Button } from '@automattic/components';
+interface Props {
+	goNext: () => void;
+}
+
+const Intro: React.FC< Props > = ( { goNext } ) => {
+	return (
+		<div className="link-in-bio-intro__content">
+			<h1 className="link-in-bio-intro__title">
+				Hello!
+				<span>Let’s set up your</span>
+				Link in Bio
```

Comment:
Remember about translations, look at how `@wordpress/react-i18n` is used in other steps 😄